platform-coop-toolkit/coop-map-directory-index
# feat: allow organization profile to be edited after creation

## Description

This PR adds the ability for user to edit organizations which they administer.

## Steps to test

1. Create an organization.
2. Visit the organization's profile.
3. Try editing via "Edit basic information", "Edit overview", and "Edit contact information" links (the last is found on the "Contact" tab.

**Expected behavior:** All fields are editable.

## Additional information

Not applicable.

## Related issues

Depends on #126.
